deepset_cloud_sdk/_api/config.py

```diff
@@ -40,7 +40,7 @@ def load_environment() -> bool:
         "No environment variables were loaded from the .env file. Create the .env file and add API_KEY and API_URL there, or run `deepset-cloud login` in the terminal to let SDK create the file for you."
     )
 
-# connection to deepset Cloud
+# connection to deepset AI Platform
```

Comment:
great that you migrated that one! Let's do a followup pr with a global replace later :)